static/app/views/feedback/feedbackListPage.tsx

```diff
@@ -53,13 +55,14 @@ const LayoutGrid = styled('div')`
 
   height: 100%;
   width: 100%;
-  padding: ${space(2)} ${space(4)};
+  padding: 0 ${space(4)} ${space(2)} ${space(4)};
```

Comment:
if we leave this alone then would we still need the marginTop?

having margin-top on the banner means we'll have to revisit this line when the banner gets replaced

Reply:
yes because layout gap has the margin top, and when the feedback banner is dismissed it leaves an extra gap :( so i made up for it by removing the margin top in the layout gap

Reply:
Oh, it's because the grid has a fixed number of rows :(